### PR TITLE
Fixed grammar and typos for GAPS

### PR DESCRIPTION
**Pull request by Pound-Hash**
- Fixed typo
- Fixed grammar and typos for GAPS

### CODE REVIEW

File: sway/sway.5.scd [hunk 1]
```diff
@@ -284,7 +284,7 @@ set|plus|minus|toggle <amount>
 *resize* shrink|grow width|height [<amount> [px|ppt]]
 	Resizes the currently focused container by _amount_, specified in pixels or
 	percentage points. If the units are omitted, floating containers are resized
-	in px and tiled containers by ppt. _amount_ will default to 10 if omitted.
+	in px and tiled containers by ppt. _Amount_ will default to 10 if omitted.
```
Comment (emersion) [inline sway/sway.5.scd:287]: As stated in the other PR, please leave this change out

Comment (Pound-Hash) [reply sway/sway.5.scd:287]: Sorry, I'm new to this.

File: sway/sway.5.scd [hunk 2]
```diff
@@ -668,12 +668,12 @@ The default colors are:
...
 
 *gaps* inner|outer|horizontal|vertical|top|right|bottom|left <amount>
-	Sets default _amount_ pixels of _inner_ or _outer_ gap, where the inner
-	affects spacing around each view and outer affects the spacing around each
-	workspace. Outer gaps are in addition to inner gaps. To reduce or remove
-	outer gaps, outer gaps can be set to a negative value. _outer_ gaps can
-	also be specified per side with _top_, _right_, _bottom_, and _left_ or
-	per direction with _horizontal_ and _vertical_.
+	Sets the default _amount_ of pixels for the _inner_ or _outer_ gaps, where _inner_
+	affects spacing around each view and _outer _affects spacing around each
```
Comment (somelazykoala) [inline sway/sway.5.scd:672]: \_outer_ instead of _outer _

File: sway/sway.5.scd [hunk 3]
```diff
@@ -668,12 +668,12 @@ The default colors are:
...
+	workspace. Values for outer gaps may serve to increase or reduce those for inner gaps. To reduce or remove
+	gaps around views or workspaces, outer gaps can be set to a negative value. _Outer_ gaps can
```
Comment (somelazykoala) [inline sway/sway.5.scd:674]: Here too, I think it is preferable to use lowercase ```outer``` instead of ```Outer```

Comment (Pound-Hash) [reply sway/sway.5.scd]: I made the requested changes. Is it okay now?

Comment (somelazykoala) [reply sway/sway.5.scd]: LGTM!

**Pull request closed without merging**